clients/go/commands/activateJobs_command.go

```diff
@@ -0,0 +1,88 @@
+package commands
+
+import (
+	"context"
+	"github.com/zeebe-io/zeebe/clients/go/pb"
+	"github.com/zeebe-io/zeebe/clients/go/utils"
+	"io"
+	"time"
+)
+
+type DispatchActivateJobsCommand interface {
+	Send() ([]*pb.ActivatedJob, error)
+}
+
+type ActivateJobsCommandStep1 interface {
+	JobType(string) ActivateJobsCommandStep2
+}
+
+type ActivateJobsCommandStep2 interface {
+	Amount(int32) ActivateJobsCommandStep3
+}
+
+type ActivateJobsCommandStep3 interface {
+	DispatchActivateJobsCommand
```

Comment:
What does this?

Reply:
https://golang.org/doc/effective_go.html#embedding

Reply:
Thanks :+1: